File: test/end-to-end/Classes/Statics/static functions.rb
#
class Utilities
    def self.get_longest(words)
        longest = ""

        for word in words
            if word.length > longest.length
                longest = word
            end
        }

        Utilities.log(longest)

        return longest
    end

    def self.log(word)
        puts "Logging: " + word
    end
end
#

File: test/end-to-end/Classes/Statics/static functions.ts
//
class Utilities {
    public static getLongest(words: string[]): string {
        let longest: string = "";

        for (let word of words) {
            if (word.length > longest.length) {
                longest = word;
            }
        }

        Utilities.log(word);

        return longest;
    }

    public static log(word: string): void {
        console.log("Logging: " + word);
    }
}
//
